bundle/edu.gemini.dataman.app/src/main/java/edu/gemini/dataman/osgi/Activator.java

```diff
@@ -4,80 +4,125 @@
+    // the new things to work without trying to start any of the old Data Man.
+    // Actually I may just convert the whole thing to Scala.
 
-    private final Set<Principal> _user = Collections.<Principal>singleton(StaffPrincipal.Gemini());
-    private final List<BundleActivator> _delegates = new ArrayList<>();
+//    private OsgiDatamanContext _dmanContext;
+//    private WorkingStoreTracker _wsTracker;
+//    private HttpTracker _httpTracker;
+//
+//    private final Set<Principal> _user = Collections.<Principal>singleton(StaffPrincipal.Gemini());
+//    private final List<BundleActivator> _delegates = new ArrayList<>();
+
+    private String requiredProp(BundleContext ctx, String propName) {
+        final String prop = ctx.getProperty(propName);
+        if (prop == null) {
+            LOG.warning("Data Manager property `" + propName + "` missing.  Data Manager not started.");
+            throw new RuntimeException("Missing " + propName + " in app configuration.");
+        }
+        return prop;
+    }
 
     public void start(BundleContext ctx) throws Exception {
         LOG.fine("start dataman-app bundle");
 
-        _delegates.add(new edu.gemini.datasetfile.osgi.Activator());
-        _delegates.add(new edu.gemini.dirmon.impl.osgi.Activator());
-        _delegates.add(new edu.gemini.datasetrecord.osgi.Activator());
-        for (BundleActivator ba: _delegates)
-            ba.start(ctx);
+        final String gsaHost = requiredProp(ctx, GSA_HOST);
+        LOG.info("Data Manager GSA host is " + gsaHost);
 
-        _httpTracker = new HttpTracker(ctx);
-        _httpTracker.open();
+        final String gsaAuth = requiredProp(ctx, GSA_AUTH);
 
-        _dmanContext = new OsgiDatamanContext(ctx, _user);
-        _dmanContext.start();
+        final Site site = SiteProperty.get(ctx);
+        if (site == null) {
+            LOG.warning("Data Manager could not determine the site.  Not started.");
+            throw new RuntimeException("Missing or not parseable " + SiteProperty.NAME + " property.");
+        } else {
+            LOG.info("Data Manager site is " + site.displayName);
+        }
 
-        RawCopier rc = _registerRawCopier(ctx, _dmanContext);
-        String path = _dmanContext.getConfig().getWorkDir().getPath();
-        _wsTracker = new WorkingStoreTracker(ctx, path, rc);
-        _wsTracker.start();
-    }
+        final Dictionary<String, Object> dict = new Hashtable<>();
+        dict.put(COMMAND_SCOPE, "gsa");
+        dict.put(COMMAND_FUNCTION, new String[]{"gsa"});
 
-    private static RawCopier _registerRawCopier(BundleContext ctx, DatamanContext dctx) {
-        DatamanConfig conf = dctx.getConfig();
-        LOG.fine("Register DirListener for: " + conf.getRawDir());
+        // TODO: we don't have a GS GSA test server or GS data so fix to GN for now
+        ctx.registerService(GsaCommands.class, GsaCommands$.MODULE$.apply(gsaHost, Site.GN, gsaAuth), dict);
```

Comment:
There's a GN test server and I've asked for a GS test server.  In production it will use production servers at GN and GS.  So probably 4 different GSA servers.